### ISSUE TITLE
Can't launch VSCode after latest update

### ISSUE THREAD

**Issue by esulu** [microsoft/vscode#78856]
- VSCode Version: 1.37.0
- OS Version: 10.0.17763

Steps to Reproduce:
1. run `code --version`

Result:

`Fatal error in , line 0`
`Check failed: (i_isolate->snapshot_blob()) == nullptr.`
`FailureMessage Object: 0000004D6D4FF578`

I normally launch VSCode using a Windows shortcut on my start menu, however after not launching I tried with powershell and it still didn't work.

**Comment by bpasero** [comment]
Can you try to reproduce with our nightly insider builds? You can give our preview releases a try from: https://code.visualstudio.com/insiders/

**Comment by Lajnold** [comment]
I have the same problem with 1.37.0. Rolling back to 1.36.1 solves it. The nightly insider also works.

**Comment by Lajnold** [comment]
1.37.0 started working now too. I use the portable zip version. I replace my previous installation each time I upgrade, keeping the data/ folder. After some back and forth with 1.36.1, insider, and 1.37.0, it started working. This time, I also started up the unzipped 1.37.0 before moving it to replace the old installation, and without the old data/ folder. It still works after replacing the old installation while keeping the data/ folder. I don't know which step solved it.

**Comment by esulu** [comment]
Update: I got VSCode working again thanks to @Lajnold 's suggestions. I'm going to list my steps below in case anyone else is having the same problem.

-  I downloaded the 64bit .zip version from https://code.visualstudio.com/Download

- I made a new folder for VSCode (mine was under C:/Users/user/AppData/Local/Programs/) and extracted the .zip file (without touching the old folder)

- Ran Code.exe and it worked!

Keep in mind that it does indeed keep your settings/extensions
